testing/gyp/APKBUILD

```diff
@@ -21,4 +21,9 @@ package() {
 	python2 setup.py install --root="$pkgdir"
 }
 
-sha512sums="bfe8484290e7216888c001638a5b6e42fc254ecf13320c9398a1c2a71e968f8338aebe2fe6edc64de3fe75203ad704bd8976caef21affe21f97430b838fefc52  aca1e2c3d346d704adfa60944e6b4dd06f4728be.tar.gz"
+cleanup_srcdir() {
+	[ -d src ] && chmod -R +w src
```

Comment:
Any reason why this is necessary ? as far as i am aware the only ones that need it are go packages that use modules because thay are made read-only.

Reply:
For unknown reason I got same problem as you mention for go packages during several unsuccessful builds of this package. I have not dig to much in this issue, just solve it this way.